build/Versions.props

```diff
@@ -69,4 +66,6 @@
     <ShouldlyVersion>3.0.0</ShouldlyVersion>
   </PropertyGroup>
 
-</Project>
+  <Target Name="GetVsixVersion" />
```

Comment:
fyi: this target was added in the newer version of the RoslynTools.RepoToolset package. For current branch we should override it to avoid issues with the execution.